.isort.cfg

```diff
@@ -1,2 +1,2 @@
 [settings]
-known_third_party = matplotlib,numpy,pyro,setuptools,sphinx_rtd_theme,torch
+known_third_party = gpytorch,matplotlib,numpy,pyro,setuptools,sphinx_rtd_theme,torch
```

Comment:
Not sure about this... @jacobrgardner ?

Reply:
For some reason the `pre-commit` hook is forcing me to add this as per below:
```
Joes-Mac:gpytorch-fork Joe$ git commit -am 'Revert spurious .isort.cfg and mark SpectralMixtureKernel as stationary'
Flake8...................................................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Forbid new submodules....................................................Passed
Mixed line ending........................................................Passed
Trim Trailing Whitespace.................................................Passed
Debug Statements (Python)................................................Passed
black....................................................................Passed
seed isort known_third_party.............................................Failed
hookid: seed-isort-config

Files were modified by this hook.

isort....................................................................Passed
Check file encoding......................................................Passed
Non-executable shell script filename ends in .sh.....(no files to check)Skipped
Forbid binaries......................................(no files to check)Skipped
CRLF end-lines checker...................................................Passed
No-tabs checker..........................................................Passed
```

Reply:
I _suspect_ if we consistently used relative imports from within the `gpytorch` package this would go away. I think this would be good to do anyway.

Reply:
^ could related to: https://github.com/pre-commit/pre-commit/issues/842

Reply:
My best guess is it is treating the absolute gpytorch imports from the unit tests as "third party" because they don't live anywhere in the gpytorch module itself.

For now, I think this is mostly harmless relative to the benefits we get from having isort run consistently on the package -- just yesterday isort helped me fix half a dozen bad relative imports that were only working properly because of the order they appeared in.

Reply:
I agree that `isort` is great. And this `pre-commit` embedding in git is awesome - I'm going to use it in my projects!